src/rust/ide/view/src/project.rs

```diff
@@ -57,8 +58,34 @@ impl Model {
     }
 
     fn set_style(&self, is_light:bool) {
-        if is_light { self.app.themes.set_enabled(&["dark"])  }
-        else        { self.app.themes.set_enabled(&["light"]) }
+        if is_light { self.set_dark_style()  }
+        else        { self.set_light_style() }
+    }
+
+    /// Sets style of IDE to the one defined by parameter `theme`.
+    pub fn set_style_by_theme(&self, theme: ensogl_theme::Theme) {
+        match theme {
+            ensogl_theme::Light => { self.set_light_style() },
+            ensogl_theme::Dark  => { self.set_dark_style() }
+        }
+    }
+
+    fn set_light_style(&self) {
+        ensogl_theme::builtin::light::enable(&self.app);
+        self.set_html_style("light-theme");
+    }
+
+    fn set_dark_style(&self) {
+        ensogl_theme::builtin::dark::enable(&self.app);
+        self.set_html_style("dark-theme");
+    }
+
+    fn set_html_style(&self, style:&'static str) {
+        let root_elem = web::get_element_by_id("root");
+        match root_elem {
+            Ok(v)  => v.set_class_name(style),
+            Err(_) => self.logger.warning("Failed to find root."),
```

Comment:
Write it this way please:
```rust
web::get_element_by_id(&self.logger,"root",|root| root.set_class_name(style));
```

The function probably doesnt exist in `web` but it will be crazy helpful in many places, so add it there pls.